test_mcp_server.py

```
@@ -0,0 +1,172 @@
+    
+    if content_length is None:
+        return None
+    
+    # Read content
+    content = process.stdout.read(content_length).decode('utf-8')
+    return json.loads(content)
+
+def test_mcp_server():
+    """Test the MCP server functionality."""
+    print("Testing ASR-GoT MCP Server...")
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)